bundles/org.openhab.binding.modbus.studer/src/main/resources/ESH-INF/thing/BSP-channel-types.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0" xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- Power Channel Type -->
+	<channel-type id="Pbat">
+		<item-type>Number:Power</item-type>
+		<label>Power</label>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<!-- State of charge Channel Type -->
+	<channel-type id="SOC">
+		<item-type>Number:Dimensionless</item-type>
+		<label>State of charge</label>
```

Comment:
Words in labels should be capitalized. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions